_component_design/table-pagination.md

```diff
@@ -0,0 +1,303 @@
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct a pagination bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-pagination/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+The pagination bar is centered below the table.
+
+Refer to these spacing specifications when you construct a pagination bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-pagination/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Behavior
+
+Use the following behavior guidelines when you construct a pagination bar.
+
+- When the first page is selected, the **First Page** and **Back** buttons are visible and disabled.
+- When the last page is selected, the **Next** and **Last Page** buttons are visible and disabled.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-pagination/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Row count control specifications
+
+Refer to the following specifications when you construct row count controls.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+Row count controls contain the following elements:
+
+- **Rows (required)**: Rows is a text label placed to the left of the row count controls.
+- **Row count controls (required)**: The user can click a row count control value to change the number of records displayed on a page. The number of records can be determined at implementation, and largely depends on the performance of the back-end system serving the data.
```

Comment:
BP: row control value should be whole numbers distributed rationally and multiples of 5. Don't do 10, 20, 42. Also don't do 6.5, 13, and 19.5. Good - 10, 25, 50. This pattern helps record load time.

Reply:
Done.